boards/xtensa/esp32s3_devkitm/Kconfig.defconfig

```diff
@@ -18,3 +18,6 @@ config HEAP_MEM_POOL_SIZE
 choice BT_HCI_BUS_TYPE
 	default BT_ESP32 if BT
 endchoice
+
+config ESP_SPIRAM
+	default y
```

Comment:
I think SPIRAM should be enabled in the application code rather on board level.

Reply:
Would be great if if would be selected automatically based on devicetree node corresponding to SPIRAM.